### PR TITLE
dcrwallet: advice --walletpass use on err when no passphrase is given.

### LINKED ISSUE #930: Advise user to use --promptpublicpass
**Issue** : 
When user try to open a secured wallet (with a public password) without using `--walletpass` or `--promptpublicpass`, dcrwallet respond with :
```
[ERR] DCRW: Open failed: invalid passphrase for master public key
```

In this case, the user know there is a problem related to a "master public key" but do not know how to solve it.

**Proposed solution** : 
Advise user to use either `--promptpublicpass` or `--walletpass`.
```
[INF] DCRW: Try again with --promptpublicpass
```

### PR DESCRIPTION
**Pull request by dnldd**
resolves #930.

### CODE REVIEW

**Comment by jrick** [comment]
We should catch this error from the main package and log the additional help instead of wrapping the error with cli options.  The loader is used by other subsystems (like grpc) and the additional error context may not make sense.

**Comment by dnldd** [comment]
Updated.

File: dcrwallet.go [hunk 1]
```diff
@@ -177,6 +178,14 @@ func run(ctx context.Context) error {
 		w, err := loader.OpenExistingWallet(walletPass)
 		if err != nil {
 			log.Errorf("Failed to open wallet: %v", err)
+			appErr := err.(apperrors.E)
+			if appErr.ErrorCode == apperrors.ErrWrongPassphrase {
```
Comment (jrick) [inline dcrwallet.go]: use `apperrors.IsError(err, apperrors.ErrWrongPassphrase)` instead which won't panic if the error is not an `apperrors.E`.

File: dcrwallet.go [hunk 2]
```diff
@@ -177,6 +178,14 @@ func run(ctx context.Context) error {
...
+				// walletpass not provided, advice using --walletpass or --promptpublicpass
+				if len(walletPass) == 0 {
+					log.Errorf("Retry with --walletpass or --promptpublicpass.")
```
Comment (jrick) [inline dcrwallet.go]: Don't think this should be an error log, perhaps info instead.

**Comment by dnldd** [comment]
Thanks for the review, changes have been made.

**Comment by jrick** [comment]
@CruzMolina thanks, this will be resolved in the release.  Fix is on the release-1.2 branch.

**Pull request merged**